devops/jobs/SandboxTermination.groovy

```diff
@@ -39,6 +39,7 @@ class SandboxTermination{
                     file('AWS_CONFIG_FILE','tools-edx-jenkins-aws-credentials')
                     string('ROLE_ARN', "launch-sandboxes-role-arn")
                     string('HIPCHAT_API_KEY', "hipchat-api-key")
+                    string('EDX_GIT_BOT_TOKEN', "edx_git_bot_token")
```

Comment:
This is good, you'll also need to pass it to the command (using your new argument from https://github.com/edx-ops/sysadmin/pull/161/files#diff-5c57fef0a092fa561b5c98fe8dede2e8R91)

https://github.com/edx/jenkins-job-dsl/blob/af129202b9c798a1b93d716b4d3a077dce7c0f19/devops/resources/sandbox-termination.sh